lib/modules/manager/circleci/__fixtures__/config3.yml

```diff
@@ -1,18 +1,16 @@
-aliases:  
+aliases:
   - &nodejs
     image: cimg/node:14.8.0
 
 version: 2
 jobs:
-  checkout: 
-    <<: *defaults
```

Comment:
why remove?

Reply:
YAML parsing fails because there there is no definition for the `defaults` anchor.

Reply:
ok, but is it a valid circleci file? do they provide an implicit default anchor?

we could simply strip such anchors before parsing, like we do in gitlab manager

Reply:
I don't thinks so, I can't find any reference to something like default anchor which gets injected into the file before parsing. 

See https://discuss.circleci.com/t/using-defaults-syntax-in-config-yaml-aka-yaml-anchors/16168 for example.

The other test files contain anchors too and parsing fine. 
Actually the fact anchors are now resolved is the reason I had to change the tests.